concepticondata/conceptlists.tsv

```diff
@@ -463,3 +463,4 @@ Rogic-2013-346	Rogić, Maja and Jerončić, Ana and Bošnjak, Marija and Sedlar,
 Dunabeitia-2022-500	Duñabeitia, Jon Andoni and Baciero, Ana and Antoniou, Kyriakos and Antoniou, Mark and Ataman, Esra and Baus, Cristina and Ben-Shachar, Michal and Çağlar, Ozan Can and Chromý, Jan and Comesaña, Montserrat and Filip, Maroš and Filipović Đurđević, Dušica and Gillon Dowens, Margaret and Hatzidaki, Anna and Januška, Jiří and Jusoh, Zuraini and Kanj, Rama and Kim, Say Young and Kırkıcı, Bilal and Leminen, Alina and Lohndal, Terje and Yap, Ngee Thai and Renvall, Hanna and Rothman, Jason and Royle, Phaedra and Santesteban, Mikel and Sevilla, Yamila and Slioussar, Natalia and Vaughan-Evans, Awel and Wodniecka, Zofia and Wulff, Stefanie and Pliatsikas, Christos	2022		500	naming test, basic	English	American English, Australian English, Basque, Western Flemish, British English, Catalan, Cypriot Greek, Czech, Finnish, French, German, Greek, Hebrew, Hungarian, Italian, Korean, Lebanese Arabic, Malay, Malaysian English, Mandarin Chinese, Dutch, Norwegian, Polish, Portuguese, Quebec French, Rioplatense Spanish, Russian, Serbian, Slovak, Spanish, Turkish, Welsh, global 	https://figshare.com/articles/dataset/Untitled_Item/19328939	Dunabeitia2022		This is a continuation of the [Dunabeitia et al. (2018)](:bib:Dunabeitia2018)dataset, now with 32 languages (American English, Australian English, Basque, Western Flemish, British English, Catalan, Cypriot Greek, Czech, Finnish, French, German, Greek, Hebrew, Hungarian, Italian, Korean, Lebanese Arabic, Malay, Malaysian English, Mandarin Chinese, Dutch, Norwegian, Polish, Portuguese, Quebec French, Rioplatense Spanish, Russian, Serbian, Slovak, Spanish, Turkish, Welsh) in total and with familiarity ratings for 500 chosen picture stimuli of the original 750. The pictures were standardized for name agreement and visual complexity as well.		
+DellaRosa-2010-417	Della Rosa, Pasquale A. and Catricà, Eleonora and Vigliocco, Gabriella and Cappa, Stefano F.	2010		417	ratings	English	Italian	https://doi.org/10.3758/BRM.42.4.1042	DellaRosa2010		This list of 417 Italian nouns was rated concerning mode of acquisition, concreteness, imageability, familiarity, age of acquisition, context availability, and abstractness. Ratings were given by 250 Italian native speakers.	1042-1048	
```

Comment:
The column with "pages" only applies if the data is given directly in an article as a table or appendix. You can delete the page numbers since the data are represented in a separate file.